arches_vue_utils/src/arches_vue_utils/utils.ts

```diff
@@ -66,3 +67,14 @@ export const getItemLabel = (
             rankLabel(a, preferredLanguageCode, systemLanguageCode),
     )[0];
 };
+
+export const getDescriptors = (
+        data: Descriptors,
+        systemLanguageCode: string,
+    ): [string, string] => {
+    if (!data) {
+        return ["", ""];
+    }
+    const descriptors = data[systemLanguageCode];
+    return [descriptors.name, descriptors.description];
```

Comment:
2 thoughts:

1 - should this be a dict? that way consumer knows what each value is instead of inferring from the return or looking at the function itself

2- does this have nullguarding? If `name` or `description` is falsey it would return `undefined` instead of a string.